### PR TITLE
media changes

### PR DESCRIPTION
**Pull request by tpaluc**
breakpoints sm,md and lg ready

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -217,9 +218,12 @@ <h2 class="heading__minor heading__minor--getstarted">
     </section>
     <section class="section section--journey">
         <div class="container container--journey">
+            <div class="container--journey-heading">
+                <div class="container--journey-develop">
             <h1 class="heading__primary heading__primary--journey">
```
Comment (zniszcz) [inline src/index.html]: indentation problem

File: src/index.html [hunk 2]
```diff
@@ -217,9 +218,12 @@ <h2 class="heading__minor heading__minor--getstarted">
...
                 Empowering Your Development Journey
             </h1>
+            <div class="journey__number-box">
             <div class="journey__numbers">
```
Comment (zniszcz) [inline src/index.html]: another indentation problem

File: src/sass/components/_box.scss [hunk 3]
```diff
@@ -2,15 +2,52 @@
...
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+    }
 }
 
 .box {
+    $journey-border-color: #e2e2e2;
+    $journey-border: 0.5px solid $journey-border-color;
+    $journey-width-breakpoint-md: 40vw;
```
Comment (zniszcz) [inline src/sass/components/_box.scss:32]: breakpoints should be global - you should avoid creating different breakpoints

File: src/sass/components/_container.scss [hunk 4]
```diff
@@ -28,4 +41,9 @@
     &--faq {
         background-color: $bg-light;
     }
+    &--journey-heading {
+        @media (min-width: $breakpoint-lg) {
+        display: flex;
```
Comment (zniszcz) [inline src/sass/components/_container.scss:46]: another indentation problem

File: src/sass/layout/_body.scss [hunk 5]
```diff
@@ -1,20 +1,66 @@
...
+    &__buttons {
+        @media (min-width:$breakpoint-lg) {
+            align-self: flex-end;
+        }
+    }
+    &__img {
+        display: none;
+        @media (min-width: $breakpoint-lg) {
+        display: block;
+        max-width: 90vw;
```
Comment (zniszcz) [inline src/sass/layout/_body.scss]: magic number

File: src/sass/layout/_header.scss [hunk 6]
```diff
@@ -1,4 +1,14 @@
     .heading {
+        $examples-breakpoint-md-width: 50vw;
+        $welcome-breakpoint-lg-width: 60vw;
+
```
Comment (zniszcz) [inline src/sass/layout/_header.scss]: 2x empty line

**Review by zniszcz — CHANGES_REQUESTED** [review]
Please pay attention to indents and magic numbers.

File: src/sass/components/_box.scss (hunk 3 above)
Comment (tpaluc) [reply src/sass/components/_box.scss:32]: So, it's not okay to put different values for different breakpoints. 
Better should be global like: 
$width-xs: 10vw;
$width-sm: 20vw;
$width-md: 30vw;
$width-lg: 40vw;

Comment (zniszcz) [reply src/sass/components/_box.scss:32]: yep

**Review by zniszcz — APPROVED** [review]

**Pull request merged**